lib/atomic/misc.less

```diff
@@ -267,7 +267,7 @@
 //  $  OUTLINE
 //  ----------------------------------------------------------------------------
 .outline-none { outline: 0 !important; }
-.outline-ring { outline: solid var(--su-static4) var(--focus-ring); }
+.outline-ring { outline: solid var(--su-static4) var(--translucent-secondary); }
```

Comment:
I think the doc for this atomic class will need to be updated:
https://stackoverflow.design/product/base/outline

Furthermore we might want to put a disclaimer that `outline-ring` should not be used to highlight focus states because it is not WCAG compliant. Essentially saying that this atomic class is for  "decorative-only" use and should not convey any semantic meaning.

Reply:
If it was not for the breaking changes I would even go as far as suggesting to rename the class to something like `.outline-decorative` or `.outline-decorative-ring`

Reply:
> I think the doc for this atomic class will need to be updated

Currently, it still mimics our focus style (though it references `--translucent-secondary`). We'll probably want it to mimic our focus style after we update to the new style broadly. Considering this, 

> If it was not for the breaking changes I would even go as far as suggesting to rename the class to something like .outline-decorative or .outline-decorative-ring

Ultimately, instead of renaming I'd like to get rid of this atomic class altogether. We reference it in only [a single place](https://github.com/StackEng/StackOverflow/blob/231517c19550c525ed2477285d7884ecea6e5c81/StackOverflow/_Scripts/PostCollections/CreateEditPage.ts#L177) and I think it's a class has very little utility.

**Related**: Every once in a while, I come across some minor breaking change I think would be worth including in the next major release whenever it comes along. Do you think it's reasonable to add Jira issues for these scenarios with a tag like `breaking-change` that way we could find all proposed breaking changes when planning for a major release?

Reply:
> Ultimately, instead of renaming I'd like to get rid of this atomic class altogether. We reference it in only [a single place](https://github.com/StackEng/StackOverflow/blob/231517c19550c525ed2477285d7884ecea6e5c81/StackOverflow/_Scripts/PostCollections/CreateEditPage.ts#L177) and I think it's a class has very little utility.

Oh yes, let's remove that. :) 

> Related: Every once in a while, I come across some minor breaking change I think would be worth including in the next major release whenever it comes along. Do you think it's reasonable to add Jira issues for these scenarios with a tag like breaking-change that way we could find all proposed breaking changes when planning for a major release?

That makes sense to me, once we have few of those we could start planning a major release. 
Feel free to go ahead with that. Maybe add also the label `stacks-classic` since in our backlog we have tickets for several Stacks libraries these days.

Reply:
I've created [STACKS-540](https://stackoverflow.atlassian.net/browse/STACKS-540) for removal of `.outline-ring`